core/services/relay/evm/method_binding.go

```diff
@@ -115,15 +115,15 @@ func (m *methodBinding) blockNumberFromConfidence(ctx context.Context, confidenc
 		return nil, err
 	}
 
-	latest, finalized, err := m.ht.LatestAndFinalizedBlock(ctx)
+	_, finalized, err := m.ht.LatestAndFinalizedBlock(ctx)
 	if err != nil {
 		return nil, err
 	}
 
 	if confirmations == evmtypes.Finalized {
 		return big.NewInt(finalized.Number), nil
 	} else if confirmations == evmtypes.Unconfirmed {
-		return big.NewInt(latest.BlockNumber()), nil
+		return nil, nil
 	}
```

Comment:
Could we do this in a separate PR just so that Ilja / relayer owner could stamp it?

Reply:
done here-> https://github.com/smartcontractkit/chainlink/pull/13906